firestore/ios/firestore-snippets-cpp/AppDelegate.mm

```diff
@@ -17,17 +17,23 @@
 #import "AppDelegate.h"
 #import <Firebase/Firebase.h>
 #include "firebase/app.h"
+#include "firebase/firestore.h"
 
 @interface AppDelegate ()
 
 @end
 
 @implementation AppDelegate
 
-
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
   [FIRApp configure];
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-variable"
+  // [START get_firestore_instance]
   firebase::App::Create();
+  auto db = firebase::firestore::Firestore::GetInstance();
```

Comment:
Question: what's the reason to prefer explicitly qualifying all the namespaces?

Reply:
I just wanted this code to be copy-pasteable. Seeing as the overwhelming majority of users will already have `using firebase::firestore::Firestore` at the top of the file, this is probably unnecessary.